profiles/schema.py

```diff
@@ -1646,6 +1695,18 @@ class Mutation(graphene.ObjectType):
         "* `CONNECTED_SERVICE_DELETION_FAILED_ERROR`: The profile deletion failed for one or more connected services."
     )
     # TODO: Add the complete list of error codes
+    delete_my_service_data = DeleteMyServiceDataMutation.Field(
+        description="Deletes the data of the profile which is linked to the currently authenticated user from one "
+        "connected service.\n\n"
+        "Requires authentication.\n\nPossible error codes:\n\n"
+        "* `PROFILE_DOES_NOT_EXIST_ERROR`: Returned if there is no profile linked to "
+        "the currently authenticated user.\n"
+        "* `MISSING_GDPR_API_TOKEN_ERROR`: No API token available for accessing the service.\n"
+        "* `SERVICE_CONNECTION_NOT_FOUND`: The user is not connected to the service\n"
```

Comment:
Small documentation typo: it's SERVICE_CONNECTION_NOT_FOUND_ERROR.

I was also thinking if this should be named in a similar way to the PROFILE_DOES_NOT_EXIST_ERROR, so SERVICE_CONNECTION_DOES_NOT_EXIST_ERROR.

Reply:
Changed.